distribution/build.gradle

```diff
@@ -238,6 +238,10 @@ configure(subprojects.findAll { ['archives', 'packages'].contains(it.name) }) {
         from { project(':distribution:tools:plugin-cli').jar }
         from { project(':distribution:tools:plugin-cli').configurations.runtime }
       }
+      into('tools/security-cli') {
```

Comment:
I am pretty sure this is not where we want to add this since the comment makes me think this section also applies to the oss distribution which does not have xpack.

Reply:
Thanks, that makes sense. I can take advantage of the `oss` boolean and call `with libFiles(oss)` where applicable so that it only includes the jars in `tools/security-cli` is `oss` is false

Reply:
Also, I think this can probably just go under `tools` now that I see where we are putting the jar. My suggestion for this directory yesterday was due to my outdated view of how files are laid out within the distribution.

Reply:
Hmm...I think that `tools/security-cli` is the right place? It mimics what we did with the `plugin-cli`.

Reply:
derp. Ignore me and leave it as is